azurerm/internal/services/network/resource_arm_application_gateway.go

```diff
@@ -1276,6 +1276,21 @@ func resourceArmApplicationGateway() *schema.Resource {
 				},
 			},
 
+			"firewall_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
```

Comment:
Is ID the only property in this block? if so it might make sense to just have one property `firewall_policy_id` for simplicity. YDYT?

Reply:
Makes sense. Done in cd7d32c23.